src/components/KRadio/KRadio.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <label
+    :checked="isSelected"
     class="k-radio"
-    :class="$attrs.class"
+    :class="[isTypeDefault ? 'k-radio-default' : `k-radio-${type}`, $attrs.class ? $attrs.class : '']"
```

Comment:
question: can't you just append `$attrs.class` regardless of whether it is empty?

Reply:
If it's empty it was rendering as `class="undefined"` which isn't pretty, that's why I added this check